lib/stencil-bundle.js

```diff
@@ -49,6 +49,7 @@ class Bundle {
         this.options = options;
 
         this.templatesPath = path.join(themePath, 'templates');
+        this.externalTemplatesPath = path.join(themePath, 'node_modules/bc-ui-lib/templates');
```

Comment:
maybe it should come from config?
Have you thought about supporting multiple templates libraries?

Reply:
done